readme.md

```diff
@@ -36,7 +37,7 @@ And so much more! Check the [usage section](#usage) for more.
 
 *If in case, you have firewalls or other traffic-blockers, add learningOrchestra as an exception.*
 
-Ex: In Google Cloud Platform each of the VMs must allow both http and https traffic.
+Ex: In Google Cloud Platform each of the VMs must allow both `allow_http` and `allow_https` traffic rules.
```

Comment:
Ex: In Google Cloud Platform each of the VMs must allow both http and https traffic.
I think `allow_https` is understood from the above sentence and it sounds better.

Reply:
@navendu-pottekkat I agree with your suggestions, please make the changes, sorry my bad english.